packages/hint-doctype/src/hint.ts

```diff
@@ -41,8 +41,28 @@ export default class implements IHint {
                 const matched = doctypeRegExp.exec(line);
 
                 if (matched) {
+                    let endLine, endColumn;
+
+                    const doctypeText = matched[1];
+
+                    if (doctypeText) {
+                        endColumn = matched.index;
```

Comment:
nit: remove these lines and do the assignment where the vars are declared on line 44.